tests/slow_tests/slow_tests.cpp

```diff
@@ -47,6 +47,7 @@
 #include <currency/currency.wast.hpp>
 #include <exchange/exchange.wast.hpp>
 #include <infinite/infinite.wast.hpp>
+#include "memory_test/memory_test.wast.hpp"
```

Comment:
once memory_test is moved to the `contracts/` tree, this should become `#include <memory_test/memory_test.wast.hpp>`

Reply:
See my previous response.